pkg/skaffold/deploy/helm/args.go

```diff
@@ -201,3 +204,51 @@ func (h *Deployer) installArgs(r latest_v1.HelmRelease, builds []graph.Artifact,
 
 	return args, nil
 }
+
+// sanitizeFilePath is used to sanitize filepaths that are provided to the `setFiles` flag
+// helm `setFiles` doesn't work with the unescaped filepath separator (\) for Windows or if there are unescaped tabs and spaces in the directory names.
+// So we escape all odd count occurrences of `\` for Windows, and wrap the entire string in quotes if it has spaces.
+// This is very specific to the way helm handles its flags.
+// See https://github.com/helm/helm/blob/d55c53df4e394fb62b0514a09c57bce235dd7877/pkg/cli/values/options.
+// Otherwise the windows `syscall` package implements its own sanitizing for command args that's used by `exec.Cmd`.
+// See https://github.com/golang/go/blob/6951da56b0ae2cd4250fc1b0350d090aed633ac1/src/syscall/exec_windows.go#L27
+func sanitizeFilePath(s string, isWindowsOS bool) string {
+	if len(s) == 0 {
+		return `""`
+	}
+	needsQuotes := false
+	for i := 0; i < len(s); i++ {
+		if s[i] == ' ' || s[i] == '\t' {
+			needsQuotes = true
+			break
+		}
+	}
+
+	if !isWindowsOS {
+		if needsQuotes {
+			return fmt.Sprintf(`"%s"`, s)
+		}
+		return s
+	}
+
+	var b []byte
+	slashes := 0
+	for i := 0; i < len(s); i++ {
+		switch s[i] {
+		case '\\':
+			slashes++
+		default:
+			// ensure a single slash is escaped
+			if slashes == 1 {
```

Comment:
Do we need to handle a trailing slash? Or two trailing slashes?

Reply:
I don't think so. We're expecting these to end in filenames.

Reply:
Can we validate that these values are files?  Then we should be good to go.

Reply:
I need to use my Windows machine to test. Will try and do it tonight.

Reply:
`helm` will fail with the right validation errors. I don't think we should be doing another validation.